src/cart/crt_iv.c

```diff
@@ -1019,8 +1019,8 @@ crt_ivf_bulk_transfer(struct crt_ivns_internal *ivns_internal,
 		D_ERROR("Bulk transfer failed; rc=%d\n", rc);
 
 		output->ifo_rc = rc;
-		rc = crt_reply_send(rpc);
-		D_ASSERT(rc == 0);
+		/* Reply can fail */
+		crt_reply_send(rpc);
```

Comment:
do we need to at least log the failure?

Reply:
carts internal code already does it along with error code received from mercury. I only printed error in one place where childs rpc was involved to help with some future debug if needed